packages/fixie-sdk/src/request-wrapper.tsx

```diff
@@ -4,6 +4,7 @@ import { ShowConversation, ConversationHistoryContext } from 'ai-jsx/core/conver
 import { Json } from './json.js';
 import { FixieConversation } from './conversation.js';
 import { OpenAI, OpenAIClient, ValidChatModel } from 'ai-jsx/lib/openai';
```

Comment:
will need to rev fixie-sdk as well

Reply:
Done. Updated package versions (`ai-jsx` and `fixie-sdk`) and Changelog.